Code/GraphMol/Kekulize.cpp

```diff
@@ -538,85 +546,94 @@ void KekulizeFragment(RWMol &mol, const boost::dynamic_bitset<> &atomsToUse,
       dummyAts[atom->getIdx()] = 1;
     }
   }
-  if (!foundAromatic) {
+  if (!foundAromatic || atomsToUse.none()) {
```

Comment:
`atomsToUse` isn't modified before this point, we can move the `atomsToUse.none()` check to the top of the function to exit early if we don't have any atoms that can be used.